docs/examples/polytope.ipynb

```diff
@@ -54,146 +59,800 @@
+       "               &#x27;2024-12-04 18:00:00&#x27;, &#x27;2024-12-05 00:00:00&#x27;],\n",
+       "              dtype=&#x27;datetime64[ns]&#x27;, name=&#x27;t&#x27;, length=145, freq=None))</pre></div></li></ul></div></li><li class='xr-section-item'><input id='section-9ee6c31c-ba3a-45b7-a72b-2e445b3d97a9' class='xr-section-summary-in' type='checkbox'  checked><label for='section-9ee6c31c-ba3a-45b7-a72b-2e445b3d97a9' class='xr-section-summary' >Attributes: <span>(8)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'><dt><span>class :</span></dt><dd>od</dd><dt><span>Forecast date :</span></dt><dd>2024-11-20T00:00:00Z</dd><dt><span>domain :</span></dt><dd>g</dd><dt><span>expver :</span></dt><dd>0001</dd><dt><span>levtype :</span></dt><dd>sfc</dd><dt><span>number :</span></dt><dd>1</dd><dt><span>stream :</span></dt><dd>enfo</dd><dt><span>type :</span></dt><dd>pf</dd></dl></div></li></ul></div></div>"
       ],
       "text/plain": [
-       "  centre shortName    typeOfLevel  level  dataDate  dataTime stepRange  \\\n",
-       "0   ecmf         z  isobaricInhPa      1  20200915         0         0   \n",
-       "1   ecmf         z  isobaricInhPa      1  20200915         0        12   \n",
-       "\n",
-       "  dataType  number gridType  \n",
-       "0       fc       0       sh  \n",
-       "1       fc       0       sh  "
+       "<xarray.Dataset> Size: 176kB\n",
+       "Dimensions:   (x: 1, y: 1, z: 1, number: 50, datetime: 1, t: 145)\n",
+       "Coordinates:\n",
+       "  * x         (x) float64 8B -9.104\n",
+       "  * y         (y) float64 8B 38.81\n",
+       "  * z         (z) int64 8B 0\n",
+       "  * number    (number) int64 400B 1 2 3 4 5 6 7 8 9 ... 43 44 45 46 47 48 49 50\n",
+       "  * datetime  (datetime) <U20 80B '2024-11-20T00:00:00Z'\n",
+       "  * t         (t) datetime64[ns] 1kB 2024-11-20 ... 2024-12-05\n",
+       "Data variables:\n",
+       "    tcc       (x, y, z, number, datetime, t) float64 58kB 0.9792 1.0 ... 0.04614\n",
+       "    2t        (x, y, z, number, datetime, t) float64 58kB 296.5 296.2 ... 295.9\n",
+       "    ssrd      (x, y, z, number, datetime, t) float64 58kB 0.0 0.0 ... 3.242e+08\n",
+       "Attributes:\n",
+       "    class:          od\n",
+       "    Forecast date:  2024-11-20T00:00:00Z\n",
+       "    domain:         g\n",
+       "    expver:         0001\n",
+       "    levtype:        sfc\n",
+       "    number:         1\n",
+       "    stream:         enfo\n",
+       "    type:           pf"
       ]
      },
-     "execution_count": 1,
+     "execution_count": 10,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "import earthkit.data\n",
+    "ds.to_xarray()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f58a63b8",
+   "metadata": {},
+   "source": [
+    "## Visualise \n",
```

Comment:
This should be a chapter inside "Time series" but "Visualise"  is now on the same level.